lib/bridge/IrcHandler.js

```diff
@@ -83,6 +83,14 @@ function IrcHandler(ircBridge, config) {
         }
     );
 
+    /*
+    One of:
+    "on" - Defaults to enabled, users can choose to disable.
+    "off" - Defaults to disabled, users can choose to enable.
+    "force-off" - Disabled, cannot be enabled.
+     */
```

Comment:
My Java OCD wants to tear this comment block apart, but I won't force you to do that :p